### PR TITLE
Add advanced shiny workshop details

### PR DESCRIPTION
**Pull request by gadenbuie**
Here are the workshop details for the advanced Shiny for R workshop.

### CODE REVIEW

File: workshops/shiny_r_adv.qmd [hunk 1]
```diff
@@ -1,35 +1,35 @@
 ---
-title: Advanced to Shiny for R
+title: Building Brilliant Shiny Apps in R
```
Comment (gadenbuie) [inline workshops/shiny_r_adv.qmd]: @mine-cetinkaya-rundel I started to wonder if the word _advanced_ in "Advanced Shiny for R" might lead to some people we'd want to reach to self-select out of the workshop. So this is another take at the title.


That said, I can also see that we have plenty of prior art and maybe even some "brand" recognition for _Advanced Shiny_, so I'd be just as fine sticking with that title.

```suggestion
title: Advanced Shiny for R
```

Comment (mine-cetinkaya-rundel) [reply workshops/shiny_r_adv.qmd]: I don't think we have to use "advanced", there used to be a curriculum known by that name but we haven't taught it in some time. I think something that describes a bit  -- that one is called "Getting Started with Shiny for R". "Building" sounds like "from scratch", like you'd be first starting off. This is more of an "improving" workshop. I'll keep thinking too but maybe this helps you come up with a slightly different title?

Comment (gadenbuie) [reply workshops/shiny_r_adv.qmd]: Maybe we could just drop "Building"?

* Brilliant Shiny Apps in R

Here are a few other titles I've brainstormed

* Beyond the Basics with Shiny in R
* Scalable Shiny in R
* Scaling Shiny Apps in R
* Level Up with Shiny in R
* Mastering Shiny in R

Do any of these sound interesting/compelling/like a good fit?

Comment (mine-cetinkaya-rundel) [reply workshops/shiny_r_adv.qmd]: I like level up and mastering, and happy with either! Feel free to pick the one you prefer.

Comment (gadenbuie) [reply workshops/shiny_r_adv.qmd]: After some internal discussion with the team, we've decided we like _Level Up with Shiny for R_. I'll update the PR 😄

**Pull request merged**